courses/migrations/0016_enrollment_order.py

```diff
@@ -30,19 +30,25 @@ def get_order(user, product):
         CourseRunEnrollment.objects.filter(order=None).order_by("id").iterator()
     )
     for enrollment in run_enrollments:
-        product = Product.objects.get(
-            content_type__model="courserun", object_id=enrollment.run.id
-        )
+        try:
+            product = Product.objects.get(
+                content_type__model="courserun", object_id=enrollment.run.id
+            )
+        except Product.DoesNotExist:
+            log.exception("Product does not exist for courserun %d", enrollment.run.id)
         order = get_order(enrollment.user, product)
```

Comment:
I think this needs an `if product else None` now, same for line 49, or better yet roll that logic into `get_order`

Reply:
Added `continue` statements in catch blocks, the kwargs for retrieving the product differ depending on what type of enrollment it is.

Reply:
ah ok, makes sense